jupyterlab/README.md

```diff
@@ -16,7 +16,7 @@ To run Q# in Jupyter notebooks, remember to also install the `qsharp` package: [
 
 ## Support
 
-For more information about the Azure Quantum Development Kit, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
```

Comment:
Switching a few links from the generic "Azure Quantum" landing page to the QDK specific page.